older/check_md_raid.py

```diff
@@ -113,7 +113,7 @@ def test_raid(verbosity):
                 state = line.split(":")[-1][1:-1]
                 state = state.rstrip()
         re_clean = re.compile('^clean|active(,.*)?$')
-        if not re_clean.match(state) and state != "active":
+        if re_clean.match(state) and state != "active":
```

Comment:
This works for me:
```suggestion
        if re_clean.match(state) and state != "clean" and state != "active":
```

But then I think this would trip it up `clean, no-errors` (I need to check the other code)...